src/index.html

```diff
@@ -1016,7 +1033,7 @@ <h2 id="variantInfo"  data-magellan-target="variantInfo" class="padding-bottom-l
             if (parArr) {
                 for (var i = 0; i < parArr.length; i++) {
                     parr = parArr[i].split("=");
-                    data.id = parr[0];
+                    data.id = parr[0].replace(/-/g, " ");

```

Comment:
Does this translate the URLs with dashes (such as `Variant-Browser`) to URLs with white spaces (`Variant Browser`)?

Reply:
No in the URLs you'll see as Variant-Browser

Reply:
other changes are updated